tock/employees/models.py

```diff
@@ -130,6 +131,14 @@ def is_billable(self):
         """
         return self.billable_expectation > Decimal(0.0)
 
+    @cached_property
+    def billable_expectation(self):
+        """
+        Returns the employee's billable expectation, expressed as a percentage
+        of their hours worked
+        """
+        return Decimal(self.expected_billable_hours / settings.HOURS_IN_A_REGULAR_WORK_WEEK)
```

Comment:
Should we take advantage of the ability to set precision on Decimal in order to avoid repeating decimals? Or, since it's hidden from users, is the precision no problem?

Reply:
Yeah, I think we should 👍 . On that note - would it be useful to still display the `billable_expectation` in the admin pages (alongside the `expected_billable_hours`)?

Reply:
I think just the expectation in hours. That's how everyone thinks of them.